sphinx_search/static/js/rtd_sphinx_search.js

```diff
@@ -340,7 +312,7 @@ const generateSingleResult = (resultData, projectName) => {
             </h2> \
         </a>';
 
-    let page_link = `${resultData.link}${DOCUMENTATION_OPTIONS.FILE_SUFFIX}`;
+    let page_link = READTHEDOCS_DATA.canonical_url + resultData.full_path;
```

Comment:
Any reason for this change? I don't know if we should be forcibly changing domains in search results, it feels less than ideal.

Reply:
No... I forgot to update this PR.
This change was done when I got confused with serving fiels from nginx